src/store/Workspaces/index.ts

```diff
@@ -288,6 +288,9 @@ export const actionCreators: ActionCreators = {
       await WorkspaceClient.restApiClient.stop(workspaceId);
     } catch (e) {
       dispatch({ type: 'RECEIVE_ERROR' });
+      if (e.response?.data?.message) {
+        throw new Error(`Failed to stop the workspace, ID: ${workspaceId}, ` + e.response.data.message);
+      }
       throw new Error(`Failed to stop the workspace, ID: ${workspaceId}, ` + e);
```

Comment:
Is it going to be displayed as `Failed to stop the workspace, ID: workspace123sadfsd, Object{object}`? If yes, then let's rework it in the way to display the response code and print the object in the browser console. Maybe it's better to do pretify the error on the Workspace Client level

Reply:
it's displayed as [object Object]
![Screenshot_20201215_111928](https://user-images.githubusercontent.com/5887312/102195591-7064cb80-3ec7-11eb-97a6-7b13538888e4.png)

The reliable way to reproduce it:
1. Open Dashboard next.
2. Scale down Che:
```bash
oc patch deployment/che \
        --patch "{\"spec\":{\"replicas\":0}}
```
3. Try to perform the test operation
The image I used `sleshchenko/che-server:dshb-next-96`

Reply:
I want to add: it is a common problem for our error messages. I will prefer to implement this logic as a separate function and move it to the separate file in ```src/services```